google-cloud-firestore/src/main/java/com/google/cloud/firestore/Transaction.java

```diff
@@ -51,6 +51,16 @@
     T updateCallback(Transaction transaction) throws Exception;
   }
 
+  /**
+   * User callback that takes a Firestore Transaction
```

Comment:
Please update to mention the async nature of this callback.

Reply:
check it, please